Adds new integration [basbruss/adaptive-cover]

basbruss opened the pull request:
<!--
DO NOT REQUEST REVIEWS, THAT IS JUST RUDE, IF YOU DO THE PULL REQUEST WILL BE CLOSED!
Make sure to check out the guide here: https://hacs.xyz/docs/publish/start
-->
## Checklist

<!-- Do not open a pull request before you have completed all these, it will be closed. -->

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

<!-- Do not open a pull request before you have provided all these, it will be closed. -->

Link to current release: <https://github.com/basbruss/adaptive-cover/releases/tag/v1.4.4>
Link to successful HACS action (without the `ignore` key): <https://github.com/basbruss/adaptive-cover/actions/runs/7694819607>
Link to successful hassfest action (if integration): <https://github.com/basbruss/adaptive-cover/actions/runs/7694819596>

tjorim commented:
Be aware you do the 'HACS Validation' twice: once in the hacs.yaml, once in the validate.yml workflow. One of them is wrong (hint: the ignore key).

basbruss commented:
> Be aware you do the 'HACS Validation' twice: once in the hacs.yaml, once in the validate.yml workflow. One of them is wrong (hint: the ignore key).

Thanks! I removed the one in validate.yml and renamed it to hassfest.yml.

ludeeus commented on integration:
```diff
@@ -96,6 +96,7 @@
   "bacco007/sensor.waternsw",
   "barban-dev/homeassistant-midea-dehumidifier",
   "barleybobs/homeassistant-ecowater-softener",
+  "basbruss/adaptive-cover",
```
https://github.com/basbruss/adaptive-cover/actions/runs/11426903428/job/31790406035 hassfest is failing.

basbruss replied:
That's linked to this PR to core https://github.com/home-assistant/core/pull/128965

basbruss replied:
New release removes code that failed hassfest
